src/content/whats-new/2022/03/whats-new-03-28-futurestack-22.md

```diff
@@ -0,0 +1,41 @@
+---
+title: 'FutureStack is back—in-person at the Cosmopolitan of Las Vegas' 
+summary: 'Join us May 17-19 for inspiring keynotes, hands-on labs, technical breakouts, 
+and amazing entertainment, Vegas style.' 
+releaseDate: '2022-03-28' 
+learnMoreLink: 'https://www.futurestack.com/anqQXE?RefId=WHATSNEW' 
+---
+
+![FutureStack22](./images/FutureStack.jpg "FutureStack22")
+
+
+**Join us for FutureStack May 17-19 at the Cosmopolitan of Las Vegas**
+
+Reconnect face-to-face and node-to-node—and find out what your most innovative peers are doing.
```

Comment:
@Alanjbaptista this may need a little reword; the 'node-to-node-and'.

Reply:
updating the source file